hack/testing/logging.sh

```diff
@@ -439,8 +470,21 @@ os::build:wait_for_end "test"
 wait_for_app "test"
 ### test app added ###
 
+### kibana setup - router account, router, kibana user ###
+oc create serviceaccount router -n default
+oadm policy add-scc-to-user privileged system:serviceaccount:default:router
+oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:default:router
+oadm router --create --namespace default --service-account=router \
+     --credentials $MASTER_CONFIG_DIR/openshift-router.kubeconfig
+oc login --username=kibtest --password=kibtest
+oc login --username=system:admin
```

Comment:
can we add the os::cmd::expect_success back to this line? If we aren't doing `SETUP_ONLY` we'd need to fail out here since the rest of the test probably won't work if we can't log in as 'system:admin'